### PR TITLE
Close Sentry.io

### PR DESCRIPTION
**Pull request by Urmi-Jana**
Based on #1075

### CODE REVIEW

**Comment by parth-commits** [comment]
if its closed, i think you should remove the link

File: README.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ And we're back! Use this repo to share and keep track of software, tech, CS, PM,
 | [Intuit](https://jobs.intuit.com/job/-/-/27595/33672146000) | Mountain View, California; San Diego, California; Plano, Texas; New York, New York; Atlanta, Georgia | Software Engineer Intern |
 | [Uber](https://university-uber.icims.com/jobs/116847/job) | Various | 2023 Software Engineer Internship |
 | [Hudson River Trading](https://www.hudsonrivertrading.com/careers/?_4118765=Internship) | New York, Austin, Chicago | Algorithm Development, Software Engineering (C++ or Python) |
-| [Sentry.io](https://sentry.io/careers/internships/) | San Francisco, Toronto, Vienna | **SF and Toronto closed** Software Engineer, Intern (Summer 2023) |
+| [Sentry.io](https://sentry.io/careers/internships/) | San Francisco, Toronto, Vienna | **🔒 Closed 🔒** Software Engineer, Intern (Summer 2023) |
```
Comment (Zmwang622) [inline README.md]: please remove the link, thanks!

**Review by Zmwang622 — CHANGES_REQUESTED** [review]

**Comment by Urmi-Jana** [comment]
Removed the link. Thanks for pointing it out!

**Comment by Zmwang622** [comment]
Appreciate it!

**Pull request merged**